LICENSE.txt

```diff
@@ -1,31 +1,31 @@
-Unless noted below, Minecraft Forge, Forge Mod Loader, and all 
+Unless noted below, Minecraft NeoForge, NeoForge Mod Loader, and all 
```

Comment:
The project name is NeoForge, not `Minecraft NeoForge`, and the mod loader is Fancy Mod Loader.

Reply:
Should be fixed with newest commit, thanks!